### ISSUE TITLE
Homepage Crash and Reload on Safari

### ISSUE THREAD

**Issue by mozfet** [mozfet/ExpertBox#133]
When opening home route on Safari, it causes a crash reset cycle, and eventually, the browser gives up. Does not happen on Chrome, nor on PROD, seems an issue with the current local build.

**Comment by mozfet** [comment]
It seems the crash is caused by the materialize responsive video in the home view html. When this markup is commented out, it loads fine...
```
<video class="responsive-video" controls>
  <source src="videos/expert_box_how_it_works.mp4" type="video/mp4">
</video> -->
```

**Comment by mozfet** [comment]
Trying to replicate using [JSFiddle](https://jsfiddle.net/mozfet/y2egmvLo/4/), but not succeeding...

**Comment by mozfet** [comment]
I still do not know what the problem is, but fixed it with quite an elaborative workaround. I noticed that videos in iFrames work ok, so I created a client side route for video urls and rendered a frame with the url as source. The route renders an video template that displayes the html5 video; bonus is that this is a nice cinematic way to share videos without all the regular website trimmings. [ExpertBox.com/video/expert_box_how_it_works](ExpertBox.com/video/expert_box_how_it_works)

To make things worse, Chrome video player does not support clicking on the video to start play, while Safari does, so an event handler was added to the template to start on click of video.
